### ISSUE TITLE
Show key_count in the name of namespaces in the tree

### ISSUE THREAD

**Issue by dmagliola** [redis/RedisDesktopManager#3470]
It'd be really useful if we could have, in the "namespaces" / "folders", the number of keys under them in the label.

Something like this:

This needs to be recursive, but since it seems like all the keys are loaded in the tree already, and trees are most likely not that deep, it shouldn't be a huge performance hit (I believe)

```
- a_key
- cache (6)
  - models (5)
    - users (2)
      - cache:models:users:1
      - cache:models:users:2
    - images (3)
      - cache:models:images:1
      - cache:models:images:2
      - cache:models:images:3
  - others (1)
     - cache:others:something
- some_other_key
```

I tried adding this myself, but can't get the project to build and run in my local machine.

**Comment by uglide** [comment]
@dmagliola please see my comment in Gitter chat. Thanks!

**Comment by uglide** [comment]
Implemented by @dmagliola in 0.8.0-release
